### ISSUE TITLE
Instabug 10.7.1+ provokes debugger warning

### ISSUE THREAD

**Issue by florentilroxxe** [Instabug/Instabug-iOS#428]
Hello, 

We updated our Instabug pod recently from 10.7.0 to 10.7.1 (problem has also been noticed on 10.7.4). As soon as we updated our debugger started to print the warning you can read below. Any idea as to what might has caused this? 

### Steps to Reproduce the Problem

pod install Instabug 10.7.1+ (10.7.0 and below were fine)
Try to use debugger po command on any object in our project

### Expected Behavior

Debugger prints object

### Actual Behavior

Debugger prints : 

```
warning: Swift error in fallback scratch context: error: failed to load module 'xxxx'

note: This error message is displayed only once. If the error displayed above is due to conflicting search paths to Clang modules in different images of the debugged executable, this can slow down debugging of Swift code significantly, since a fresh Swift context has to be created every time a conflict is encountered.
```

### Instabug integration code
*Make sure to remove your app token.*

### SDK Version

10.7.1

**Comment by andresinaka** [comment]
I'm experimenting the same problem. I had to downgrade to `10.5.2` and everything is back to normal.

**Comment by husseinkishk** [comment]
Hi @florentilroxxe, @andresinaka 
This is an issue in lldb, which happens sometimes when you are trying to integrate Instabug in a Swift-only project.

A workaround for this issue is to add Objective-C files and a bridging header to your project, like mentioned in this [StackOverflow post](https://stackoverflow.com/questions/54776459/whats-the-solution-for-error-couldnt-irgen-expression-no-additional-error/61824142#61824142).

Also please make sure you're using the latest version 10.7.1 before trying the workaround.

**Comment by hassaanelgarem** [comment]
@florentilroxxe We think this could be caused by the xcode version we use for generating the binary. We created the following custom build using a newer xcode version:
`pod 'Instabug', :podspec => 'https://ios-releases.instabug.com/custom/fix-bump-xcode/Instabug.podspec'`

@andresinaka You can find the same custom build for SPM on the following branch: `fix/bump-xcode` 

Can you guys try this and let us know if it fixes the issue? 🙏

**Comment by andresinaka** [comment]
@hassaanelgarem I just tested the branch `fix/bump-xcode` on my project and I still have the same issue.

<img width="283" alt="bild" src="https://user-images.githubusercontent.com/1957233/124718791-c0a88700-df06-11eb-9c7d-9ea18d755e60.png">

```
(lldb) po subtitle
warning: Swift error in scratch context: error: failed to load module 'xxx'
.
Shared Swift state for xxx has developed fatal errors and is being discarded.
REPL definitions and persistent names/types will be lost.

error: expression failed to parse, unknown error
```

As soon as I go back to `10.5.2` the problem gets fixed and the debugger goes back to life.

**Comment by hassaanelgarem** [comment]
@andresinaka That's unfortunate to hear. I just pushed a new build on branch `downgrade-xcode`. This build is generated using the same xcode version that `10.5.2` was generated with.

It would be great if you can give this one a try as well 🙏

**Comment by andresinaka** [comment]
@hassaanelgarem good news!!!! The `downgrade-xcode` branch is working just fine 😄 🎉 🎉 🎉 

<img width="302" alt="bild" src="https://user-images.githubusercontent.com/1957233/124920288-781cc680-dff7-11eb-86d8-efbf5dba0ff8.png">

how/where do I request my "I helped Instabug" t-shirt???? 🤣 🤣

**Comment by tbajis** [comment]
@hassaanelgarem I'm experiencing the same issue with versions `10.7.1+` (I'm currently using `10.7.2`). 

I can confirm that this is not an issue for version `10.7.0` as I tested downgrading. However, I was using `10.7.2` to because of  a [crash fix](https://github.com/Instabug/Instabug-iOS/issues/418#issuecomment-863817557) that it brings in. Is there a recommended solution for this issue that doesn't involve downgrading?

**Comment by hingarn** [comment]
@husseinkishk I am experiencing the same issue. StackOverflow post does not resolve the problem (the post did resolve https://github.com/Instabug/Instabug-iOS/issues/427).

**Comment by hassaanelgarem** [comment]
@florentilroxxe @andresinaka @tbajis @hingarn This issue has been resolved in the latest release `10.7.5`, so sorry for the inconvenience 🙏

**Comment by andresinaka** [comment]
Confirming `10.7.5` works like a charm! 🎈🎈🎈🎈🎈
